sql/export_qaqc.sql

```diff
@@ -341,3 +341,22 @@ INTO usetype_changes
 FROM prev a
 JOIN current b 
 ON a.usetype = b.usetype;
+
+-- Create QAQC tables of count of records by agency and usetype
+DROP TABLE if exists records_by_agency;
+SELECT
+"AGENCY", count(*) 
+INTO records_by_agency
+FROM colp GROUP BY "AGENCY";
+
+DROP TABLE if exists records_by_usetype;
+SELECT
+"USETYPE", count(*) 
```

Comment:
What are your thoughts on including the expanded category and/or expanded category description here? I think it might facilitate better qaqc graphs down the line by allowing easier grouping of usetypes. Probably same with the agency_usetype one

Reply:
Oh interesting. Are you talking about about the `EXPANDCAT` and `EXCATDESC` columns? Is the `EXPANDCAT` a key to look up another value?

Reply:
Yes - my understanding is expandcat is the code and excatdesc is the human readable description of the same thing: A broader cut of the same question about usetype. So each usetype only has one expandcat, and each expandcat has many usetypes. Would expand our ability to group/cut the data on the QAQC side in a way that is maybe meaningful for the user

Reply:
Ok word. I'll add expanded category (which I'll just refer to on the front-end as "category") and agency-category combinations. I'll also put them on their own page as it's a lot of content. Then we can decide which we want to keep during fine-tuning.
And I'll probably save this to-do until after you and Jingyi are done. Wrote up the corresponding issue #222